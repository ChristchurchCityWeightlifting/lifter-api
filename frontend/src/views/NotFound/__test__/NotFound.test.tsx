/** @format */

import { test, expect } from "@jest/globals";

test("Not Found", () => {
  expect(true).toBe(true);
});
