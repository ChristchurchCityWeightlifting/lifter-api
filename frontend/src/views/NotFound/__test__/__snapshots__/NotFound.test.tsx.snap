// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<div
  className="MuiBox-root css-0"
>
  <style
    dangerouslySetInnerHTML={
      {
        "__html": ".css-kyj2s4-MuiTypography-root{margin:0;font-family:"Roboto","Helvetica","Arial",sans-serif;font-weight:300;font-size:3.75rem;line-height:1.2;letter-spacing:-0.00833em;margin-bottom:0.35em;color:#1976d2;}",
      }
    }
    data-emotion="css kyj2s4-MuiTypography-root"
  />
  <h2
    className="MuiTypography-root MuiTypography-h2 MuiTypography-gutterBottom css-kyj2s4-MuiTypography-root"
  >
    Not Found
  </h2>
</div>
`;
